tools/pythonpkg/duckdb_python.cpp

```diff
@@ -294,7 +294,7 @@ PYBIND11_MODULE(DUCKDB_PYTHON_LIB_NAME, m) { // NOLINT
 
 	m.doc() = "DuckDB is an embeddable SQL OLAP Database Management System";
 	m.attr("__package__") = "duckdb";
-	m.attr("__version__") = DuckDB::LibraryVersion();
+	m.attr("__version__") = std::string(DuckDB::LibraryVersion()).substr(1);
```

Comment:
I think this should go entirely in the future - since this still couples the Python package version to the underlying DuckDB version. The Python package version should be passed in in a different manner so these two can deviate.